bin/setup

```diff
@@ -144,13 +148,14 @@ migrate_existing_database (){
 }
 
 setup_database (){
-  bundle exec rails db:setup
+  bundle exec rails db:setup ||
+  echo 'Unable to create the databases for you.
+Please ensure your database.yml is configured for your system and try again.'
+}
 
-  if [ $? -ne 0 ]
-  then
-    echo 'Unable to create the databases for you. Please ensure your database.yml is configured for your system and try again.'
-    exit 0
-  fi
```

Comment:
Just a small refactor for readability. Also, a successful `echo` will always return zero.